lib/rubocop/cop/style/redundant_constant_base.rb

```diff
@@ -42,6 +42,7 @@ class RedundantConstantBase < Base
         MSG = 'Remove redundant `::`.'
 
         def on_cbase(node)
+          return if lint_constant_resolution_cop_enabled?
```

Comment:
This behavior should be documented in the cop's description.

Reply:
Sure! I've update the docs of `Style/RedundantConstantBase` and `Lint/ConstantResolution` cops.